docs/pyodide/target/target_sketch.js

```diff
@@ -1629,17 +1629,7 @@ touchEnded = None
 windowResized = None
 `;
 
-let userCode = `
-def setup():
-    createCanvas(200, 200)
-
-def draw():
-    background(200)
-    diameter = sin(frameCount / 60) * 50 + 50
-    fill("blue")
-    ellipse(100, 100, diameter, diameter)
-
-`;
+let userCode = checkForSketch();
```

Comment:
We shouldn't change the `target_sketch.js` to call this function because this should be exclusively of the demo editor. My idea would be for the JS from the demo editor to be call this `checkForSketch` function and update both sketch-holder and the IDE code with that.

I know that this can prevent the sketch from automatically load itself, but I don't see a problem to request the user to click on the Run button to see the sketch running.

Reply:
Actually, I **fell** that we could update this line to just be `let userCode = ""`. This hard coded user code is probably a legacy from the very first implementations of pyodide mode

Reply:
For sure, I'll send this change, had thought of removing it as well but tried to keep things as they were for the most part.